Jenkinsfile.integration

```diff
@@ -23,7 +23,7 @@ pipeline {
     parameters {
         /* first value in the list is the default */
         choice(choices: ['airship', 'osh'], description: 'Which deployment mechanism?', name: 'deployment')
-        choice(choices: ['caasp3', 'caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
+        choice(choices: ['caasp4', 'caasp3'], description: 'Which caasp version to use?', name: 'caasp_version')
```

Comment:
Are you sure this is wise? We should still figure out some remaining problems, e.g. skuba networking

Reply:
I dont expect this to be merged anytime soon, and Im sick of every push going into jenkins, canceling the caasp3 work and launching the caasp4 job instead.

This will be faster while we iterate over it and can be dropped in a second :+1:

Reply:
OK